### ISSUE TITLE
Update version on packagist to support Symfony 7

### ISSUE THREAD

**Issue by yura3d** [tijsverkoyen/CssToInlineStyles#244]
Seems it's time to update the version on packagist because of 416a55b

Current 2.2.6 conflicts with Symfony 7.0

**Comment by redheness** [comment]
In the meantime you can add this to your composer

```json
{
    "require": {
        "tijsverkoyen/css-to-inline-styles": "dev-master#416a55b as 2.2.7"
    }
}
```

This alias the current commit of this repository as the new version. I tested it and it works on Symfony 7.

I do not recommend you to push this on production, but it will allow you to work with the new Symfony version before an official version of this package is published.

**Comment by stof** [comment]
@redheness using a commit pinning is a very bad idea as this can cause weird things to happen (as this composer legacy feature is a huge hack from the time where partial updates did not exist).
The proper solution until a release is done is to use `^2.2@dev` as version constraint, allowing to install the dev version.

**Comment by it-can** [comment]
@stof can someone tag a new version please :-)

**Comment by it-can** [comment]
@tijsverkoyen can you tag a new version please? :-)

**Comment by counteraccro** [comment]
hi,
@stof , is it possible for you to create a version 2.2.7 to take into account the latest version of the master and authorize Symfony 7
I would like to avoid going through dev if possible.
Is there any particular reason not to create a new version now?
Thanks in advance.

**Comment by yura3d** [comment]
Resolved in 83ee6f3
